pushbullet

```diff
@@ -7,6 +7,8 @@ API_URL=https://api.pushbullet.com/v2
 PROGDIR="$(cd "$( dirname "$0" )" && pwd )"
 unset QUIET
 
+CURL="curl"
+
```

Comment:
I agree, thanks. I've changed to the previous behavior (of hard-coded `curl`) and now use `$CURLOPTS` ... it's easy enough to use `curl=$(which curl)`, but that's up to the maintainer(s) and was not the intent of my PR. Pls see my recent commit.